lib/kernel/src/erl_epmd.erl

```diff
@@ -164,14 +164,9 @@ to when accepting new distribution requests.
       Host :: atom() | string() | inet:ip_address(),
       Port :: non_neg_integer().
 listen_port_please(_Name, _Host) ->
-    try
-        %% Should come up with a new name for this as ERL_EPMD_PORT describes what
-        %% port epmd runs on which could easily be confused with this.
```

Comment:
Following this comment, I thought it's the perfect time to pick a different name, that's why I named the kernel parameter `erl_epmd_listen_port`.

Reply:
`erl_epmd_node_listen_port` because this is only relevant to `erl_epmd`, but it is not the `listen_port` of `erl_epmd`, but of the node.

Reply:
Sounds good!